
  0%|                                                                                                                                                                  | 0/2997 [00:00<?, ?it/s]

  0%|                                                                                                                                                        | 1/2997 [00:05<4:17:01,  5.15s/it]

  0%|                                                                                                                                                        | 2/2997 [00:09<3:48:52,  4.59s/it]

  0%|▏                                                                                                                                                       | 3/2997 [00:12<3:24:01,  4.09s/it]

  0%|▏                                                                                                                                                       | 4/2997 [00:16<3:14:35,  3.90s/it]

  0%|▎                                                                                                                                                       | 5/2997 [00:20<3:12:05,  3.85s/it]

  0%|▎                                                                                                                                                       | 6/2997 [00:24<3:11:26,  3.84s/it]

  0%|▎                                                                                                                                                       | 7/2997 [00:28<3:16:39,  3.95s/it]

  0%|▍                                                                                                                                                       | 8/2997 [00:31<3:12:18,  3.86s/it]

  0%|▍                                                                                                                                                       | 9/2997 [00:35<3:12:46,  3.87s/it]

  0%|▌                                                                                                                                                      | 10/2997 [00:39<3:09:54,  3.81s/it]

  0%|▌                                                                                                                                                      | 11/2997 [00:43<3:06:20,  3.74s/it]

  0%|▌                                                                                                                                                      | 12/2997 [00:47<3:09:43,  3.81s/it]

  0%|▋                                                                                                                                                      | 13/2997 [00:51<3:13:53,  3.90s/it]
  0%|▋                                                                                                                                                      | 13/2997 [00:55<3:30:48,  4.24s/it]
Traceback (most recent call last):
  File "/home/mineslab-ubuntu/anaconda3/envs/DLT/lib/python3.9/site-packages/PIL/ImageFile.py", line 515, in _save
    fh = fp.fileno()
AttributeError: '_idat' object has no attribute 'fileno'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/mineslab-ubuntu/KDH_CAL2/dlt/inference.py", line 277, in <module>
    app.run(main)
  File "/home/mineslab-ubuntu/anaconda3/envs/DLT/lib/python3.9/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/home/mineslab-ubuntu/anaconda3/envs/DLT/lib/python3.9/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/home/mineslab-ubuntu/KDH_CAL2/dlt/inference.py", line 217, in main
    collage.save(save_file_name)
  File "/home/mineslab-ubuntu/anaconda3/envs/DLT/lib/python3.9/site-packages/PIL/Image.py", line 2413, in save
    save_handler(self, fp, filename)
  File "/home/mineslab-ubuntu/anaconda3/envs/DLT/lib/python3.9/site-packages/PIL/PngImagePlugin.py", line 1398, in _save
    ImageFile._save(im, _idat(fp, chunk), [("zip", (0, 0) + im.size, 0, rawmode)])
  File "/home/mineslab-ubuntu/anaconda3/envs/DLT/lib/python3.9/site-packages/PIL/ImageFile.py", line 519, in _save
    _encode_tile(im, fp, tile, bufsize, None, exc)
  File "/home/mineslab-ubuntu/anaconda3/envs/DLT/lib/python3.9/site-packages/PIL/ImageFile.py", line 538, in _encode_tile
    errcode, data = encoder.encode(bufsize)[1:]
KeyboardInterrupt